pkg/storage/db.go

```diff
@@ -1058,3 +1058,84 @@ order by ae.ts asc
 	}
 	return nil
 }
+
+// AccountOwnerStorer is an implementation of AccountOwnerStorer interface that saves account ID, its owner and champions of the account to a database
+func (db *DB) AccountOwnerStorer(ctx context.Context, accountOwner domain.AccountOwner) error {
+
+	tx, err := db.sqldb.Begin()
+	if err != nil {
+		return err
+	}
+
+	err = db.storeAccountOwner(ctx, accountOwner, tx)
+
+	if err != nil {
+		if rollbackErr := tx.Rollback(); rollbackErr != nil {
+			return errors.Wrap(rollbackErr, err.Error()) // so we don't lose the original error
+		}
+		return err
+	}
+	if err = tx.Commit(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (db *DB) storeAccountOwner(ctx context.Context, accountOwner domain.AccountOwner, tx *sql.Tx) error {
+
+	sqlStatement := fmt.Sprintf(`REPLACE INTO aws_account (account) VALUES($1)`) // nolint
+	if _, error := tx.ExecContext(ctx, sqlStatement, accountOwner.AccountID); error != nil {
+		return error
+	}
+
+	sqlStatement = fmt.Sprintf(`IF EXISTS(SELECT * from person WHERE login=$1)
+									 UPDATE person SET email=$2, name=$3, valid=$4
```

Comment:
This part seems to be missing "where" clause, and will update values for all entries in the `person` table